src/utils/work/seekWorkResponse.js

```diff
@@ -27,7 +29,7 @@ const getRemainingWorkerStartTime = (creationTimestamp) => {
 const seekFromS3 = async (ETag, experimentId) => {
   let response;
   try {
-    response = await fetchAPI(`/v1/workResults/${experimentId}/${ETag}`);
+    response = await fetchAPI(`/${config.currentApiVersion}/workResults/${experimentId}/${ETag}`);
```

Comment:
The pattern that we have currently uses `if` statements to determine the api url. Maybe better to use the same pattern and follow suit?